docs/references/strategies/README.md

```diff
@@ -105,3 +103,39 @@
 - [carthage](platforms/ios/carthage.md)
 - [cocoapods](platforms/ios/cocoapods.md)
 - [swiftPM](platforms/ios/swift.md)
+
+## Strategies by type
+
+Languages supported by FOSSA CLI can have multiple strategies for detecting dependencies, one primary strategy that yields ideal results and zero or more fallback strategies. Within this list of strategies, we have the concept of *static* and *dynamic* strategies. Static strategies parse files to find a dependency graph (example: parse a `package-lock.json` file). Dynamic strategies are required when analyzing package managers that do not offer complete lockfiles, such as Gradle or Golang. Dynamic strategies require a working build environment to operate in.
```

Comment:
[nit] The name of the language is [Go, not Golang](https://go.dev/doc/faq#go_or_golang).
```suggestion
Languages supported by FOSSA CLI can have multiple strategies for detecting dependencies, one primary strategy that yields ideal results and zero or more fallback strategies. Within this list of strategies, we have the concept of *static* and *dynamic* strategies. Static strategies parse files to find a dependency graph (example: parse a `package-lock.json` file). Dynamic strategies are required when analyzing package managers that do not offer complete lockfiles, such as Gradle or Go. Dynamic strategies require a working build environment to operate in.
```

Reply:
There are probably a lot more errors out there like this. I always type "Golang" because "Go" is honestly sometimes confusing.

Reply:
I think the court of public opinion is split https://hub.docker.com/r/circleci/golang, but I'll change the references I can find.